hamp_radar/serde_pds_files.py

```diff
@@ -14,10 +14,11 @@ def scan_pdsfile(filename: Path):
 
 
 def serialize_write_data(writefile: Path, data):
+    import orjson
```

Comment:
This seems like we need a new dependency in pyproject.toml

Reply:
yes, first we need a pyproject.toml... I'lll make serperate PR (commming soon)